datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/config.py

```diff
@@ -106,7 +106,7 @@ def config(ctx, check, sync, verbose):
                             files_failed[example_file_path] = True
                             check_display_queue.append(
                                 lambda example_file=example_file, **kwargs: echo_failure(
-                                    f'File `{example_file}` needs to be synced', **kwargs
+                                    f'File `{example_file}` is not in sync, run "ddev validate config -s"', **kwargs
```

Comment:
This should probably be on a different PR, otherwise it will mess up the changelog